kuma/javascript/src/payments/pages/management.jsx

```diff
@@ -0,0 +1,200 @@
+    const [showForm, setShowForm] = React.useState<boolean>(false);
+    const [status, setStatus] = React.useState<'success' | 'error' | 'idle'>(
+        'idle'
+    );
+    const [subscription, setSubscription] = useState<?SubscriptionData>(null);
+    const [canceled, setCanceled] = useState<?boolean>(false);
+    const userData: ?UserData = useContext(UserProvider.context);
+
+    useEffect(() => {
+        if (userData && userData.isSubscriber) {
+            const handleSuccess = (data) => {
+                const [subscription] = data.subscriptions;
+                setSubscription(subscription);
+            };
+            const handleError = () => setStatus('error');
+            getSubscriptions(handleSuccess, handleError);
+        }
+    }, [userData]);
+
+    // if there's no user data yet, don't render anything
+    if (!userData) {
+        return null;
+    }
+
+    const handleClick = (event) => {
+        event.preventDefault();
+        setShowForm(true);
+    };
+
+    const handleDeleteSuccess = () => {
+        setSubscription(null);
+        setCanceled(true);
+        setStatus('success');
+    };
+
+    const renderSuccess = () => (
+        <p className="alert success" data-testid="success-msg">
+            {successMsg}
+        </p>
+    );
+
+    const renderInvitation = () => (
+        <div className="active-subscriptions">
+            <Interpolated
+                id={gettext(
+                    'You have no active subscriptions. Why not <signupLink />?'
+                )}
+                signupLink={
+                    <a href={`/${locale}/payments/`}>{gettext('set one up')}</a>
+                }
+            />
+        </div>
+    );
+
+    const renderSubscriptions = () => {
+        if (!subscription) {
+            return false;
+        }
+        const date = new Date(subscription.next_payment_at);
+        const nextPaymentDate = formatDate(
+            locale,
+            subscription.next_payment_at
+        );
+        const lastActiveDate = formatDate(
+            locale,
+            date.setDate(date.getDate() - 1)
+        );
+        return (
+            <>
+                <p>
+                    {interpolate(
+                        'Next payment occurs on %(nextPaymentDate)s.',
+                        {
+                            nextPaymentDate,
+                        }
+                    )}
+                </p>
+                <div className="active-subscriptions">
+                    <ul>
+                        <li>
+                            <span className="label amount">
+                                {gettext('Amount')}
+                            </span>
+                            <span className="value">
+                                {/* amount is in cents, so divide by 100 to get dollars */}
+                                {formatMoney(locale, subscription.amount / 100)}
+                            </span>
```

Comment:
This looks like a good candidate for a description list. You can then have something like:

```
<dl className="active-subscriptions">
  <dt className="label amount">{gettext('Amount')}</dt>
  <dd className="value">
    {/* amount is in cents, so divide by 100 to get dollars */}
    {formatMoney(locale, subscription.amount / 100)}
  </dd>
...
</dl>
```

Reply:
Originally I had it as a description list but it was such a headache to style. I will try again tomorrow.